src/views/App/index.tsx

```diff
@@ -71,7 +71,7 @@ const AppRoutes = () => {
       {isUserSet && user.isGrtReviewer(userGroups) && (
         <SecureRoute
           path="/governance-review-team/:systemId/:activePage"
-          component={GovernanceReviewTeam}
+          render={() => <GovernanceReviewTeam />}
```

Comment:
Is there a reason for this?

Reply:
It's either this or do:
```
        <SecureRoute
          path="/governance-review-team/:systemId/:activePage"
          render={({ component }: any) => component()}
          component={GovernanceReviewTeam}
        />
```

It's related to an issue with the okta package unmounting routes.